docs/connecting.md

```diff
@@ -74,54 +72,38 @@ are plain strings:
-| `:read-only?`          | bool         | false              | Whether to initiate this connection in READ ONLY mode (see below)               |
-| `:in-stream-buf-size`  | integer      | 0xFFFF             | Size of the input buffered socket stream                                        |
-| `:out-stream-buf-size` | integer      | 0xFFFF             | Size of the output buffered socket stream                                       |
-| `:fn-notification`     | 1-arg fn     | logging fn         | A function to handle notifications                                              |
-| `:fn-protocol-version` | 1-arg fn     | logging fn         | A function to handle negotiation version protocol event                         |
-| `:fn-notice`           | 1-arg fn     | logging fn         | A function to handle notices                                                    |
-| `:use-ssl?`            | bool         | false              | Whether to use SSL connection                                                   |
-| `:ssl-context`         | SSLContext   | nil                | An custom instance of `SSLContext` class to wrap a socket                       |
-| `:so-keep-alive?`      | bool         | true               | Socket KeepAlive value                                                          |
-| `:so-tcp-no-delay?`    | bool         | true               | Socket TcpNoDelay value                                                         |
-| `:so-timeout`          | integer      | 15.000             | Socket timeout value, in ms                                                     |
-| `:so-recv-buf-size`    | integer      | 0xFFFF             | Socket receive buffer size                                                      |
-| `:so-send-buf-size`    | integer      | 0xFFFF             | Socket send buffer size                                                         |
-| `:cancel-timeout-ms`   | integer      | 5.000              | Default value for the `with-timeout` macro, in ms                               |
-| `:protocol-version`    | integer      | 196608             | Postgres protocol version                                                       |
-| `:object-mapper`       | ObjectMapper | JSON.defaultMapper | An instance of ObjectMapper for custom JSON processing (see the "JSON" section) |
-
-### Parameter notes
-
-#### Read Only Mode
-
-The `:read-only?` connection parameter does two things under the hood:
-
-1. It appends the `default_transaction_read_only` parameter to the startup
-   message set to `on`. Thus, any transaction gets started on `READ ONLY` mode.
-
-2. It prevents the `:read-only?` flag from overriding in the `with-tx`
-   macro. Say, even if the macro is called like this:
-
-~~~clojure
-(pg/with-tx [conn {:read-only? false}] ;; try to mute the global :read-only? flag
-  (pg/query conn "delete from students"))
-~~~
-
-The transaction will be in `READ ONLY` mode anyway.
```

Comment:
Personally I wouldn't like to drop out this fragment. Can you get it back please? The phrasing you proposed is too short and I'm afraid the context will be lost.